roles/edpm_container_quadlet/tasks/cleanup.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Stop and disable old standalone container for {{ edpm_container_quadlet_service }}
+  ansible.builtin.systemd_service:
+    name: "{{ edpm_container_quadlet_service }}"
+    state: stopped
+    enabled: false
+  become: true
+
+- name: Cleanup standalone service file for {{ edpm_container_quadlet_service }}
+  ansible.builtin.file:
+    path: "/etc/systemd/system/{{ edpm_container_quadlet_service }}.service"
+    state: absent
+  become: true
```

Comment:
should we do a `systemctl daemon-reload` here?

Reply:
We _could_, but I don't think the `systemd_service` module will let me just do a `daemon-reload` without specifying some unit. I would probably have to use `shell:` to do that.

Not sure we really need to though tbh, because this will run first, then the configuration task with run for the quadlet and do a `daemon-reload` as part of that.